fleetBuilder.go

```diff
@@ -165,7 +165,7 @@ func (f *FleetBuilder) SendNow() (Fleet, error) {
 		// Calculate cargo
 		cargoCapacity := 0
 		for _, ship := range f.ships {
-			cargoCapacity += Objs.ByID(ship.ID).(Ship).GetCargoCapacity() * ship.Nbr
+			cargoCapacity += Objs.ByID(ship.ID).(Ship).GetCargoCapacity(tx.GetResearch()) * ship.Nbr
```

Comment:
Two things, you are currently calling ogame server in a loop and get the same value many times.
Put that call outside the loop.

And Move the whole thing inside the following "if condition" below, since we don't need to calculate the cargo outside of the following if